src/main/java/com/applicaster/liga/statsscreenplugin/screens/home/adapters/TeamAdapter.kt

```diff
@@ -113,12 +114,15 @@ class TeamAdapter(val items: List<GroupModel.Ranking>, val context: Context?, li
         }
     }
 
-    private fun getIndicatorColor(position: Int): Int {
+    private fun getIndicatorColor(position: Int, rank: Int): Int {
+//        Rank 1: 3 (top 3 in green)
+//        Rank 2: 1 (team on position 4 in yellow)
+//        Rank 3: 2 (last two teams in red)
         return context?.let {
             when {
-                position < 3 -> ContextCompat.getColor(it, R.color.indicator_green)
-                position == 3 -> ContextCompat.getColor(it, R.color.indicator_yellow)
-                position > (itemCount - 3) -> ContextCompat.getColor(it, R.color.indicator_red)
+                position < 3 && rank == 3 -> ContextCompat.getColor(it, R.color.indicator_green)
+                position == 3 && rank == 1 -> ContextCompat.getColor(it, R.color.indicator_yellow)
+                position > (itemCount - 3) && rank == 2 -> ContextCompat.getColor(it, R.color.indicator_red)
                 else -> 0
```

Comment:
According to this logic, some of the rows of the list will not have color. 
Is this possible the first 3 rows will have rank 2 or 1? Are you sure this is correct logic?

Reply:
At the start of this condition, I've added the comment with requirements from the customer.

Reply:
@VladSumtsov The logic has updated